tests/test_preprocessing.py

```diff
@@ -191,6 +192,60 @@ def test_replace_x_y_nominal_lat_lon(dask, nans):
     assert set(replaced_ds.lat.dims) == set(["x", "y"])
 
 
+class TestReplaceXYNominalLatLon:
+    """
+    Reproduce the issue as in https://github.com/jbusecke/xMIP/issues/295 and assert that the fix
+    of https://github.com/jbusecke/xMIP/pull/296 does the job
+    """
+
+    def test_old_fails(self):
+        lons = self._get_dummy_longitude()
+        lons_parsed = self._interp_nominal_lon_old(lons)
+        if self._lons_parsed_make_sense(lons, lons_parsed):
+            raise ValueError(
+                "Parsed lons should be gibberish, but somehow the old implementation also works?"
```

Comment:
See my other comment. That will hopefully lead this to actually fail? Once that is the case I wouöd suggest to remove the "old way" logic to keep this test more compact